app/services/authentication/providers/facebook.rb

```diff
@@ -47,6 +47,13 @@ def self.sign_in_path(**kwargs)
         )
       end
 
+      def image_url
+        token = auth_payload.credentials.token
+        uid = auth_payload.uid
+        request = HTTParty.get("https://graph.facebook.com/#{uid}?fields=picture.type(small)&access_token=#{token}")
```

Comment:
These variables are made because Rubocop said my line was too long :(

Reply:
Ah ah :)

the URL could be moved in another variable, or better yet in a constant template string that gets interpolated by `format/sprintf` https://github.com/rubocop-hq/ruby-style-guide#sprintf

We use it in a few places:

https://github.com/forem/forem/blob/89e0ea79bb0cb6b7cd59b85770f14e70bcb75ddd/app/models/article.rb#L368

https://github.com/forem/forem/blob/89e0ea79bb0cb6b7cd59b85770f14e70bcb75ddd/app/services/slack/messengers/note.rb#L29-L35

Reply:
Woahhhhh neat! Was wondering if constants with interpolation was a thing!